spec/requests/assessment_request_spec.rb

```diff
@@ -17,7 +17,7 @@
       }
     end
 
-    it 'creates an assessment' do
+    specify do
```

Comment:
This test is pretty self-descriptive so maybe `specify` is an option here, but I think it's better to use it without description only in short cases like this: `specify { @arr.should_not be_empty }`. Such cases should be readable in English without any comments or descriptions. So probably it's better to use `it` or `specify` with a description here

Also, in addition to the redirect, you should test that a record was created/deleted, something like:
```ruby
it "redirects back to the user's page" do
  post user_assessments_path(user), params: valid_assessment_params
  expect(response).to redirect_to user
end

it 'creates an assessment' do
  expect do
    post user_assessments_path(user), params: valid_assessment_params
  end.to change(Assesment, :count).by(1)
end
```

 And the same for other tests in this PR